common/plugins/eu.esdihumboldt.hale.common.test/Tests.product

```diff
@@ -489,6 +486,7 @@
       <plugin id="org.eclipse.osgi.services"/>
       <plugin id="org.eclipse.osgi.util"/>
       <plugin id="org.eclipse.swt"/>
+      <plugin id="org.eclipse.swt.cocoa.macosx.x86_64" fragment="true"/>
```

Comment:
Trouble ahead?

Reply:
> Trouble ahead?

I did not understand. Should I remove it?

Reply:
Didn't it fail previously on the build server with cocoa bundles added?
If it does not fail due to that then it's fine.

Reply:
Didn't it fail previously on the build server with cocoa bundles added?
If it does not fail due to that then it's fine.

Reply:
I removed them in my workspace because I saw Red Cross on them so thought they might cause trouble. but they did not. So we can leave them.

Reply:
> I removed them in my workspace because I saw Red Cross on them so thought they might cause trouble. but they did not. So we can leave them.

Do you mean the Linux fragments that were present before and you removed in an earlier PR?
In any case, if they are not needed for the tests to run they can stay removed.